Avoid panics when calling into `dwrote`

acarl005 opened the pull request:
This PR removes some panicking code paths from the directwrite wrappers.

It builds off of servo/dwrote-rs#62 to consume the new fallible methods that result `Results` instead of panicking. Note that it needs a version bump in the `Cargo.toml` once that is published.

I also have servo/dwrote-rs#63 open so I can migrate more calls if that one lands.

mrobinson reviewed (dismissed):
Please increase the `dwrote` dependency version (might require a new release there) and also update the commit title to match the new PR title.

acarl005 commented:
I've updated the commit message per the new title. I don't think there's a published version with the required changes, so I've updated the `Cargo.toml` to point to a git revision for the moment.

jdm commented:
Waiting on https://github.com/servo/dwrote-rs/pull/65 so I can publish a new version.

jdm commented:
dwrote 0.11.3 is on crates.io.
